downstream_projects.json

```
@@ -1,8 +1,8 @@
 {
   "packageDir": "./dist",
   "projects": {
-    "sample-app-angular": "https://github.com/ui-router/sample-app-angular.git",
-    "angular17": "./test-angular-versions/v17",
-    "typescript52": "./test-typescript-versions/typescript5.2"
+    "sample-app-angular": "https://github.com/lukaVarga/ui-router-sample-app-angular.git",
```

I guess, this should be changed back to
```suggestion
    "sample-app-angular": "https://github.com/ui-router/sample-app-angular.git",
```